src/style.css

```diff
@@ -1,3 +1,106 @@
 body {
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
Consider specifying a fallback font-family in case 'Roboto' isn't available. For example: 'font-family: Roboto, Arial, sans-serif;' can be changed to 'font-family: Roboto, 'Helvetica Neue', Arial, sans-serif;' to provide a better cross-platform compatibility.